### PR TITLE
Move GoGrid to a direct REST driver

### PR DESCRIPTION
**Pull request by techhat**

### CODE REVIEW

File: salt/cloud/clouds/gogrid.py [hunk 1]
```diff
@@ -281,3 +165,375 @@ def create(vm_):
...
+
+def show_instance(name, call=None):
+    '''
+    Start a machine by name
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt-cloud -a start vm_name
```
Comment (rallytime) [inline salt/cloud/clouds/gogrid.py]: I think this code block example isn't quite right.

File: salt/cloud/clouds/gogrid.py [hunk 2]
```diff
@@ -281,3 +163,367 @@ def create(vm_):
...
```
Comment (rallytime) [inline salt/cloud/clouds/gogrid.py]: I think this code block example isn't quite right.

Comment (techhat) [inline salt/cloud/clouds/gogrid.py]: I fixed the one example, but what's wrong with this one?

Comment (rallytime) [inline salt/cloud/clouds/gogrid.py]: Nothing. I was commenting on an old page and my comment accidentally ended up here, instead of on the other one.

**Pull request merged**